src/pages/Transfer/CrossChainTransferForm/components/ChainSelect/ChainSelect.tsx

```diff
@@ -62,7 +56,7 @@ const ChainSelect = forwardRef<HTMLInputElement, ChainSelectProps>(
 
     return (
       <>
-        <Flex direction='column' flex='1' className={className}>
+        <Flex ref={ref} direction='column' flex='1' className={className}>
```

Comment:
Not sure this is applied to the correct element—React was throwing an error related to `forwardRef`

Reply:
You dont need to worry about this since this file can be replaced by implementing new Select